operator/cmd/mesh/install.go

```diff
@@ -184,12 +184,15 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 
 	// Detect whether previous installation exists prior to performing the installation.
 	exists := revtag.PreviousInstallExists(context.Background(), kubeClient)
+	rev := iop.Spec.Revision
+	if rev == "" {
+		_ = revtag.DeleteTagWebhooks(context.Background(), kubeClient, revtag.DefaultRevisionName)
```

Comment:
This introduces a (likely small) downtime that may lead to a pod not being injected. I think this is something that is never acceptable, even for a small time, as failiure to inject a pod can lead to authz policy bypass

Reply:
Hm there is already a small window of injection downtime (between calling `DeactivateIstioInjectionWebhook` and creating the generated revision tag).... will have to rework this too.

Reply:
ok, I think that is worth doing but we can merge this in the mean time if desired as its not making the situation any worse I guess

Reply:
Yeah this is meant to unblock docs test update, I think we should either merge or revert the breaking PR

Reply:
Bit of a brain dump but another thing we're trying to avoid here is having a window of double-injection by different revisions.  For example, default tag points to revision `foo`, we install without a revision so then we also have the istio-sidecar-injector webhook injecting for default labels until at the end of the installation we deactivate that webhook.

Maybe this is taping over the fact that there shouldn't be an injector for `istio-injection=enabled` installed through the `istio-discovery` chart anymore now that we control this through default tag... would fix the issue of in-cluster operator reconciling over the deactivated webhook too